frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/JsonInputParam.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { TextArea } from '@patternfly/react-core';
+import { InputParamProps } from './types';
+
+export const JsonInputParam: React.FC<InputParamProps> = ({ id, value, onChange }) => {
+  const [jsonValue, setJsonValue] = React.useState(value ? JSON.stringify(value) : '');
+
+  return (
+    <TextArea
+      id={id}
+      autoResize
+      resizeOrientation="vertical"
+      value={jsonValue}
+      onChange={(event, newValue) => {
+        setJsonValue(newValue);
+        onChange(event, newValue);
+      }}
+    >
+      <pre>{jsonValue}</pre>
```

Comment:
I dont think the TextArea component takes children, so this `pre` is not applied

Reply:
This was leftover from testing. Will remove.

Reply:
Is it just not possible then?

Reply:
i dont see support in PF for it